recipes/bamtools/build.sh

```diff
@@ -26,3 +28,6 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DZLIB_ROOT=$PREFIX \
       ..
 make install
+ar cr ${PREFIX}/lib/libbamtools_utils.a src/utils/CMakeFiles/BamTools-utils.dir/*o
```

Comment:
If the static library that ships with bamtools is missing something can you raise this upstream? That sounds like a bug.

Reply:
Hmm, it doesn't appear that bamtools is maintained any more. That could pose a problem since a number of packages depend on it.

Reply:
@dpryan79  well it seems that the package is not being maintained actively. This is because I built some functions around these "utils" but they are not making it in the bioconda repo.

Reply:
> Hmm, it doesn't appear that bamtools is maintained any more. That could pose a problem since a number of packages depend on it.

I know it does not seem maintained, maybe I can add the lib+include in new bamtools/ subdirs to make sure that there are no collisions?